roles/download/tasks/kubeadm_images.yml

```diff
@@ -22,3 +40,33 @@
 
 - name: container_download | download images for kubeadm config images
   command: "{{ bin_dir }}/kubeadm config images pull --config={{ kube_config_dir }}/kubeadm-images.yaml"
```

Comment:
We should remove this task if we are downloading them yet again in roles/download/tasks/main.yml L22

Reply:
They are not "yet again" downloaded. The current situation with this PR is:
* if download_run_once: Aggregate list of kubeadm config images, then merge these with the download list from `roles/download/defaults/main.yml` and use the download task in `roles/download/tasks/main.yml`. This is true for both `download_localhost=true` and `download_localhost=false`.
* if download_run_once is false, then the list from kubeadm images is not collected, but the above is executed to pull the images on each host.

There are some options one could consider:
* only use the main download task and never user `kubeadm config images pull` (meaning, also collect kubeadm images list and merge with default downloads when download_run_once=false)
* leave it as is (described above)
* use `kubeadm config images pull` also when `download_run_once=true` and `download_localhost=false` (because we are certain that the first master node can install kubeadm and use it as expected).

The only reason why the list is collected and downloaded with the task in main is because it may cause extra trouble to install `kubeadm` on the local ansible controller to pull the images (when `download_localhost=true`).